src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/__tests__/autocomplete.command.from.test.ts

```diff
@@ -88,12 +80,31 @@ describe('autocomplete.suggest', () => {
         await assertSuggestions('from a, b /', expected);
       });
 
+      test('on SPACE plus prefix', async () => {
```

Comment:
nit: I find this sentence difficult to understand. Can we maybe rephrase it? 🙈

Reply:
I'll do my best... 😅

Reply:
updated